.gitignore

```diff
@@ -7,3 +7,5 @@
 /tmp
 node_modules
 /coverage
+/test/hello.txt
+/fake_file.txt/random
```

Comment:
Create a directory and ignore it instead of ignoring individual files - could be a `.tmp/` or `test/.tmp` directory.